src/components/forms/Textarea/Textarea.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import classnames from 'classnames'
+
+interface TextareaProps {
+  id: string
+  name: string
+  className?: string
+  error?: boolean
+  success?: boolean
+  children?: React.ReactNode
+  inputRef?:
+    | string
+    | ((instance: HTMLTextAreaElement | null) => void)
+    | React.RefObject<HTMLTextAreaElement>
+    | null
+    | undefined
+}
+
+export const Textarea = (
+  props: TextareaProps & React.TextareaHTMLAttributes<HTMLTextAreaElement>
+): React.ReactElement => {
+  const {
+    id,
+    name,
+    className,
+    error,
+    success,
+    children,
+    inputRef,
+    ...inputProps
+  } = props
+
+  const classes = classnames(
```

Comment:
same question here on disabled and readonly as for text input.

Reply:
same as above